### DISCUSSION TITLE
How to set dark mode?

### DISCUSSION THREAD

**Discussion by tiagotobias2003** [styd/apexcharts.rb#82] [Q&A] [answered]
Hi!

I'm trying to configure the charts with dark mode, in the same way as this link:
[https://apexcharts.com/docs/options/theme/](https://apexcharts.com/docs/options/theme/)
How to make this work?

**Comment by styd** [comment, marked as answer]
Hi, @tiagotobias2003! Sorry, I missed this.
To use theme mode "dark", I guess this option should do: `theme: {mode: "dark"}`.
Let me know if this is no longer working!

**Reply by tiagotobias2003** [reply]
Hi @styd! 

don't worry, I left the question and knew it would be answered when possible, thank you very much for your help with this question.

It worked for me when I coded like this, see:

```
<% options = { title: 'My title', ytitle: 'My y title', theme: {mode: 'dark', palette: 'palette6'} } %>
<%= column_chart(@series_bar, {**options}) %>
```

Thank you very much!
